src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  let result = 0;
+  const i = Big(numberOne);
+  const j = Big(numberTwo);
```

Comment:
### [Optional]

It would be nice to use a good naming conversion.
Example: `total` and `next` instead of `i` and `j`

Reply:
done